### PR TITLE
fix(widget-config): update widget config of baseTrend and costTrend

### PR DESCRIPTION
**Pull request by yuda110**
### To Reviewers
- [ ] Skip (`style`, `chore` ONLY)
- [x] Not that difficult

### Type of Change
- [ ] New feature
- [ ] Bug fixes
- [x] Feature improvement
- [ ] Refactor
- [ ] Others (performance improvement, refactoring, CI/CD, etc.)

### Checklist
- [x] `Error / Warning / Lint / Type`

### Description
SSIA

### CODE REVIEW

File: src/services/dashboards/widgets/config.ts [hunk 1]
```diff
@@ -40,6 +38,16 @@ export const GROUP_BY = {
...
 
+export const CHART_TYPE = Object.freeze({
+    TREEMAP: 'TREEMAP',
+    MAP: 'MAP',
+    LINE: 'LINE',
+    STACKED_COLUMN: 'STACKED_COLUMN',
+    DONUT: 'DONUT',
+    PIE: 'PIE',
+    WAFFLE: 'WAFFLE',
+} as const);
```
Comment (sulmoJ) [inline src/services/dashboards/widgets/config.ts:49]: When using 'as const', you don't have to use Object.freeze!

**Review by sulmoJ — APPROVED** [review]
I left simple comments!

File: src/services/dashboards/widgets/config.ts (hunk 1 above)
Comment (yuda110) [reply src/services/dashboards/widgets/config.ts:49]: oh i missed it, thanks!

**Review by WANZARGEN — APPROVED** [review]
Nice clean code 😄

**Pull request merged**